src/routes/index.js

```diff
@@ -5,10 +5,12 @@ const router = express.Router();
 const helloWorldController = require('../controllers/hello-world');
 const heroController = require('../controllers/hero-list');
 const registrationController = require('../controllers/registrationController');
+const loginController = require('../controllers/loginController');
 
 router.get('/helloworld', helloWorldController.helloWorldController);
-
 router.get('/heroes', heroController.getHeroes);
+router.post('/login', loginController.loginController);
```

Comment:
There will be merge conflicts here, but fundamentally we have been leaving a blank line between requests to make it look pretty.